### ISSUE TITLE
add libcogl-gles2-dev to dependancy list.

### ISSUE THREAD

**Issue by WizardGed** [i-rinat/freshplayerplugin#188]
Not sure if this is a Debian sid only problem but Freshplayerplugin states glesv2 is a dependency. In Debian sid  libcogl-gles2-dev is the corresponding dependency. you may want to add that to the dependency list.

**Comment by i-rinat** [comment]
I'm using Debian testing which at the moment have the same `libcogl-gles2-dev` version as sid.

```
$ apt-file show libcogl-gles2-dev 
libcogl-gles2-dev: /usr/include/cogl/cogl-gles2/GLES2/gl2.h
libcogl-gles2-dev: /usr/include/cogl/cogl-gles2/GLES2/gl2ext.h
libcogl-gles2-dev: /usr/include/cogl/cogl-gles2/GLES2/gl2platform.h
libcogl-gles2-dev: /usr/lib/x86_64-linux-gnu/libcogl-gles2.so
libcogl-gles2-dev: /usr/lib/x86_64-linux-gnu/pkgconfig/cogl-gles2-1.0.pc
libcogl-gles2-dev: /usr/lib/x86_64-linux-gnu/pkgconfig/cogl-gles2-2.0-experimental.pc
libcogl-gles2-dev: /usr/share/doc/libcogl-gles2-dev/NEWS.gz
libcogl-gles2-dev: /usr/share/doc/libcogl-gles2-dev/README.gz
libcogl-gles2-dev: /usr/share/doc/libcogl-gles2-dev/changelog.Debian.gz
libcogl-gles2-dev: /usr/share/doc/libcogl-gles2-dev/changelog.gz
libcogl-gles2-dev: /usr/share/doc/libcogl-gles2-dev/copyright
```
As you can see, there is no file named `glesv2.pc` in that package.

```
$ apt-file search glesv2.pc
emscripten: /usr/share/emscripten/system/lib/pkgconfig/glesv2.pc
libgles2-mesa-dev: /usr/lib/x86_64-linux-gnu/pkgconfig/glesv2.pc
```
It's in `libgles2-mesa-dev` instead.

`libcogl-gles2-dev` depends on `libcogl-dev`, which in turn depends on `libgles2-mesa-dev`. That's how you've got right package installed.

**Comment by i-rinat** [comment]
I removed `libgles2-mesa-dev` from the list after inclusion of shader translator from ANGLE. Then reverted back to native GLES2 by default, leaving translator as an option. But forgot to update README file.

Fixed in 2608f9012c6ffd7cec28ac55672b57de4087e225.
